components/brave_wallet_ui/stories/locale.ts

```diff
@@ -1245,5 +1245,15 @@ provideStrings({
     'Brave works with a partner to provide this service. ' +
     'By enabling, you agree to share unsigned transactions ' +
     'and their DApp URLs with our partner.',
-  braveWalletChangeAnytimeInSettings: 'Change anytime in $1Settings$2.'
+  braveWalletChangeAnytimeInSettings: 'Change anytime in $1Settings$2.',
+
+  // Balance Details Modal
+  braveWalletAvailable: 'Available',
+  braveWalletAvailableBalanceDescription: 'Funds available for you to use.',
+  braveWalletPending: 'Pending',
+  braveWalletPendingBalanceDescription:
+    'A pending change in your wallet balance.',
+  braveWalletTotalBalanceDescription:
+    'Your available funds plus any not-yet-confirmed transactions.',
+  braveWalletUnavailableBalances: 'Some balances may be unavailable'
 })
```

Comment:
Do we need a storybook section for this new modal?

Reply:
Storybook has been added!